templates/repo/branch/list.tmpl

```diff
@@ -92,8 +99,19 @@
 											{{end}}
 										{{end}}
 									</td>
+									<td class="right aligned">
+									  {{if not .IsDeleted}}
+											<div class="ui basic jump dropdown icon button poping up" style="position: absolute; margin-left: -38px; margin-top: -18px;" data-content="{{$.i18n.Tr "repo.download_archive"}}" data-variation="tiny inverted" data-position="top right">
```

Comment:
Is it possible for you to move this (and all other) inline CSS, that you've added, into the less files?

Reply:
OK

Reply:
i tryed it - dont get it jet :(
there is only one inline: `style="position: absolute; margin-left: -38px; margin-top: -18px;"`
wich only afects  `class="ui basic jump dropdown icon button poping up"`

Reply:
i looked round about 1h, do you have any sugestions where i can read about less

Reply:
The less files are in /public/less and then are generated by `make generate-stylesheets` to put all of them into /public/css/index.css (generated). I assume you'd put your CSS additions in /public/less/_repository.less

Reply:
LESS (not sure if this is the reason for its name, but how I remembe rit) lets you type `less` than type typical CSS files since you can put selectors within selectors, a hierarchy of sort, and is a little more dynamic and has some code sytnax. Not sure if you were asking how to know more about what less is, or just read about how we do it in Gitea (my last comment will help there).

Reply:
@richmahn thanks!!

Reply:
less itself wasnt the problem, it is very intuetiv.   the `/public/less/_repository.less` was the problem.
because of an related pull i used `/public/less/_explore.less` the whole time.

for now it shuld work but in the future i rely have to now how **gitea**s less files are structured

Reply:
@techknowlogick i added a new class caled dlmenu is this ok?